python/spacetime/store_server.py

```diff
@@ -32,6 +32,9 @@ def handle_exceptions(f):
     def wrapped(*args, **kwds):
         try:
             FrameServer.app_gc_timers[kwds["sim"]] = time.time()
+            if type(args[0]) is not Register:
+                if kwds["sim"] not in FrameServer.Store.get_app_list():
+                    return {}, 401
```

Comment:
I'm using the HTTP error codes for that. Flask supports redefining the messages in JSON, but for now I just left it the default, which is expressive enough.